src/Surfnet/ServiceProviderDashboard/Domain/ValueObject/Contact.php

```diff
@@ -165,4 +165,14 @@ public function isContactSet()
 
         return true;
     }
+
+    public function isSet()
```

Comment:
What does isSet mean?

All fields are filled?
All required fields are filled?

Could the naming of the function clarify this question?

Reply:
:doh: turns out 2017 @Mkodde already created this method named `isContactSet`. I will ensure to add a comment to that method.